pkgs/applications/video/jellyfin-mpv-shim/patches/series

```diff
@@ -0,0 +1,2 @@
+disable-update-check.patch
```

Comment:
I don't think this is required at all. We have two patches here which could be easily replaced by substituteInPlace.